web/src/app/alerts/CreateAlertDialog/StepContent/step2.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Typography, Grid } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { ServiceChip } from '../../../util/Chips'
+
+const useStyles = makeStyles(theme => ({
+  hrUnderline: {
+    marginTop: 0,
+    marginBottom: 3,
+  },
+  itemTitle: {
+    paddingBottom: 0,
+  },
+  nudgeRight: {
+    marginLeft: theme.spacing(1),
+  },
+}))
+
+export default props => {
+  const { formFields } = props
+
+  const classes = useStyles()
+
+  const Item = props => (
+    <Grid item xs={12}>
+      <Typography
+        variant='subtitle1'
+        component='h3'
+        className={classes.itemTitle}
+      >
+        {props.title}
+      </Typography>
+      <hr className={classes.hrUnderline} />
```

Comment:
Material-UI has a `Divider` component you can use that will match theme colors appropriately (will be useful if we generate themes in the future, or have a dark mode)